src/main/kotlin/org/jetbrains/research/ml/ast/transformations/Plugin.kt

```diff
@@ -1,3 +1,7 @@
+/*
```

Comment:
Do we need this class? And if so, shouldn't the plugin be called 'AST-transformations' instead of 'Coding Assistant'?